### PR TITLE
[O11y][ActiveMQ] Make GA

### LINKED ISSUE #6048: Make ActiveMQ GA
The ActiveMQ integration is currently in BETA.

To make this integrations GA we need to validate the below use-case.

- [x] #8718
- [x] Can both Artemis and Classic supported.
- [x] Deprecation strategy if both cannot be supported together.

### Issues:

- [x] https://github.com/elastic/beats/issues/35113
- [ ] https://github.com/elastic/beats/issues/37543

```[tasklist]
### Issues and Enhancements
- [ ] https://github.com/elastic/integrations/issues/8861
- [ ] https://github.com/elastic/integrations/issues/8862
- [ ] https://github.com/elastic/integrations/issues/8863
- [ ] https://github.com/elastic/integrations/issues/8864
- [ ] https://github.com/elastic/integrations/issues/8865
- [ ] https://github.com/elastic/integrations/issues/8866
- [ ] https://github.com/elastic/integrations/issues/8867
```

### Enhancement Request:
https://github.com/elastic/enhancements/issues/11568
ActiveMQ released two flavours classic and Artemis with 5.18 release. Currently we support ActiveMQ classic version. As per the release highlights they mentioned that once the feature parity happens between classic and Artemis then the next major will be on Artemis.
@SubhrataK to provide updates on do we need to make ActiveMQ GA with or without supporting Artemis version.

### PR DESCRIPTION
**Pull request by harnish-elastic**
## Proposed commit message

- This PR make ActiveMQ integration GA.

## Author's Checklist

- [x] Automated Tests / Manual Tests guidelines
- [x] Timeduration guidelines
- [x] Field mapping guidelines
- [x] Document quality check
- [x] Kibana dashboard guidelines
- [x] TSDB enablement
- [x] SDHs check
- [x] Enhancement Requests check
- [x] Publish GA checklist summary

## Related issues

- Closes #6048

### CODE REVIEW

File: packages/activemq/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: 1.0.0
+  changes:
+    - description: Make ActiveMQ GA.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/1 #FIX ME
```
Comment (agithomas) [inline packages/activemq/changelog.yml:6]: Please correct PR link

**Review by agithomas — COMMENTED** [review]
@muthu-mps , can you please review the GA promotion?

**Comment by muthu-mps** [comment]
> @muthu-mps , can you please review the GA promotion?

Thanks @agithomas - I am already looking into it.

**Comment by agithomas** [comment]
As part of the GA enablement, I have created an Author checklist. Reference can be found [here](https://github.com/elastic/integrations/pull/8955). This is based on the GA guidelines document prepared by the team. 

@milan-elastic , you may refer to this if needed.

cc @lalit-satapathy , @muthu-mps

**Comment by harnish-elastic** [comment]
> As part of the GA enablement, I have created an Author checklist. Reference can be found [here](https://github.com/elastic/integrations/pull/8955). This is based on the GA guidelines document prepared by the team.


@agithomas Updated it, Thank you!

**Review by muthu-mps — APPROVED** [review]
LGTM!

**Review by aliabbas-elastic — APPROVED** [review]
LGTM

**Pull request merged**